### PR TITLE
Fix server statistics

### PR DESCRIPTION
**Pull request by KellerFuchs**
Thanks to @anowlcalledjosh for noticing the issue and sending a patch.

### CODE REVIEW

**Comment by KellerFuchs** [comment]
@hashbang/administrators Review needed.

File: static/index.html [hunk 1]
```diff
@@ -464,17 +464,17 @@
...
+5ooH7BBQ875lS+zIOFmgaxwb2NTfhMCnRgWfNQlSoVQG5HzwYbbIS4+5M2Z6a0Xy
+7cmGNqAKiuVpdj+BMB7Mfub3BLqubfvKyIFfSnBu4yTS/o2qTwbOFOfikvOOfk0h
+zaPHG3pHd9+Z4NLSLwDHlg/vpguSjMPgBJ7gxRTuXdOHF2/xfftBo9nPIWWgnwBC
+tZ5nDPoDfNCObECytmgJUm26d+b7hnwZAI39jJJKWgqPiHSn5GFsBaWU0s0+xYTR
+EeMaB8kBCPdiy0PXOmJBlu78TUq1+7ieLTsnc1mY1qcYGgIKflL5DnGwR/v9JCS1
+UPnMxxJAy36GCYf6RumLlk188ShMPN8zGX12G3QhkB8QjxXFRUCcWzPOEdL2F+Zm
+PLi4/selyMzBmuqljjbammech3odFld5qhVVsW/mvBQZd0zcwSyX+oiOqZjzUrZR
+tqofB+arFg+2lbB95lvNFSR7TcOpEGy5TVabwfj8jN91+tY/DuLnTwTmFLYpK+hW
+PJWMvdiB1o7+5JTMzh4trsv+H2yghSj/NJ5oY0LT3ESzzq3gmmE=
+=2uwg
```
Comment (DeviaVir) [inline static/index.html]: did you mean to update the key in this PR? seems unrelated

Comment (DeviaVir) [reply static/index.html]: and I think it's causing the travis build to fail now

Comment (KellerFuchs) [reply static/index.html]: It's not a key, it's a signature (that is embedded in the HTML file).
It was generated by running `make sign`, and is a separate commit in and of itself.

**Comment by KellerFuchs** [comment]
The CI failure is due to a bug in Shellcheck: it flags `printf -- ' %72s\n'` as having too few arguments.
Reported as https://github.com/koalaman/shellcheck/issues/987, so I suggest we either ignore the CI on this one or we add a rule to ignore that warning.

**Comment by KellerFuchs** [comment]
Deployed.
